zh-hant/index.md

```diff
@@ -4,13 +4,13 @@ layout: home
 version: 2.0.0
 ---
 
-Elixir School 是冀望學習和掌握 Elixir 程式語言人們的首選之地。
+Elixir School 是冀望掌握 Elixir 程式語言的朋友的首選。
 
-無論您是經驗豐富的老將，還是初來乍到的新人，都可以在課程和輔助資源中找到所需要的東西。
+無論您是老將，還是新人，都可以在課程和補充資源中找到所需的東西。
 
-通過自願譯者們的辛勤工作 Elixir School 已被翻譯成多種語言。其中一些譯文包括： [Việt ngữ][vi], [简体中文][zh-hans], [繁體中文][zh-hant], [Español][es], [Slovenčina][sk], [日本語][ja], [Polski][pl], [Português][pt], [Русском][ru], [Bahasa Indonesia][id], [Bahasa Melayu][ms], [Українською][uk], [한국어][ko], [Italiano][it], [Deutsch][de], [বাংলা][bn], [Türkçe][tr], and [ภาษาไทย][th].
+通過志願者的努力 Elixir School 已被翻譯成： [Việt ngữ][vi], [简体中文][zh-hans], [繁體中文][zh-hant], [Español][es], [Slovenčina][sk], [日本語][ja], [Polski][pl], [Português][pt], [Русском][ru], [Bahasa Indonesia][id], [Bahasa Melayu][ms], [Українською][uk], [한국어][ko], [Italiano][it], [Deutsch][de], [বাংলা][bn], [Türkçe][tr], and [ภาษาไทย][th]　。
 
-我們歡迎並鼓勵您加入我們，並期望因您加入 [elixirschool/elixirschool](https://github.com/elixirschool/elixirschool) 而繼續讓 Elixir School 變得更好！
+我們希望您能加入[我們](https://github.com/elixirschool/elixirschool) ，令 Elixir School 變得更好！
```

Comment:
Need to Translate
EN: We **welcome and encourage** you to join us **in continuing** to make Elixir School great by getting involved at ...